cypress/e2e/logout.cy.js

```diff
@@ -0,0 +1,23 @@
+describe('Logout functionality', () => {
+  const loginUser = () => {
```

Comment:
This would be better as a Cypress Command

Reply:
Thanks for the comment. It gave me a sleepless night, thinking,"how would I have done beyond what I was required to do?! Then I remembered that I was stumulated by  and inspired by Thea's comments and Elias Ekornås to do more